services/cd-service/pkg/cmd/server.go

```diff
@@ -85,7 +85,7 @@ func RunServer() {
 
 		repo, err := repository.New(ctx, repository.Config{
 			URL:            c.GitUrl,
-			Path:           "./repository",
```

Comment:
why do we change the path?
Note that this is a seperate path from "repoistory_remote".
They *should* be different from each other, otherwise, working locally is not the same as working remote.

I had an issue that was really hard to detect, because `Path` and `URL` were the same: Essentially, if they are the same, `git commit` is the same as `git push`, which is very confusing, and can only happen locally.

-> Change back to `./repository` please